src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
+import core.basesyntax.vehicles.Bulldozer;
+import core.basesyntax.vehicles.Excavator;
+import core.basesyntax.vehicles.Truck;
+
 public class MainApp {
 
+    public static void main(String[] args) {
```

Comment:
Remove the redundant empty line between the class declaration and the `main` method signature.